src/main/java/org/carlmontrobotics/commands/AlignToApriltag.java

```diff
@@ -21,18 +22,25 @@ public class AlignToApriltag extends Command {
     public final Drivetrain drivetrain;
     private Limelight limelight;
 
-    public final PIDController rotationPID = new PIDController(thetaPIDController[0], thetaPIDController[1],
-            thetaPIDController[2]);
+    public final PIDController rotationPID = new PIDController(
+                    SmartDashboard.getNumber("apriltag align kp",
+                                    thetaPIDController[0]),
+                    SmartDashboard.getNumber("apriltag align ki",
+                                    thetaPIDController[1]),
+                    SmartDashboard.getNumber("apriltag align kd",
+                                    thetaPIDController[2]));
 
     public AlignToApriltag(Drivetrain drivetrain, Limelight limelight) {
-        this.limelight = limelight;  
+            this.limelight = limelight;
         this.drivetrain = drivetrain;
         this.teleopDrive = (TeleopDrive) drivetrain.getDefaultCommand();
 
         rotationPID.enableContinuousInput(-180, 180);
         Rotation2d targetAngle = Rotation2d.fromDegrees(drivetrain.getHeading())
-                .minus(Rotation2d.fromRadians(limelight.getRotateAngleRad()));
-        rotationPID.setSetpoint(MathUtil.inputModulus(targetAngle.getDegrees(), -180, 180));
+                        .plus(Rotation2d.fromRadians(
+                                        limelight.getRotateAngleRadMT2()));
+        rotationPID.setSetpoint(MathUtil.inputModulus(targetAngle.getDegrees(),
+                        -180, 180));
         rotationPID.setTolerance(positionTolerance[2], velocityTolerance[2]);
```

Comment:
we also want to tune tolerances

Reply:
might be helpful to printout where the robot is trying to turn to for debugging purposes